toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -325,3 +330,30 @@ func enableOrDisableServices(services imagecustomizerapi.Services, baseConfigPat
 
 	return nil
 }
+
+func loadOrUnloadModules(modules imagecustomizerapi.Modules, imageChroot *safechroot.Chroot) error {
+	var err error
+
+	for _, module := range modules.Load {
+		logger.Log.Infof("Loading module (%s)", module.Name)
+		moduleFileName := module.Name + ".conf"
+		moduleFilePath := filepath.Join(imageChroot.RootDir(), "/etc/modules-load.d/", moduleFileName)
+		err = file.Write(module.Name, moduleFilePath)
+		if err != nil {
+			return fmt.Errorf("failed to write module load configuration: %w", err)
+		}
+	}
+
+	for _, module := range modules.Unload {
```

Comment:
Should `Unload` be renamed to `Disable`?

The module isn't really being "unloaded", since it is never loaded in the first place. And neither are you removing it from the "load" list but instead blocking it from being loaded at all.